common/test/acceptance/pages/lms/dashboard.py

```diff
@@ -177,3 +177,16 @@ def pre_requisite_message_displayed(self):
         Verify if pre-requisite course messages are being displayed.
         """
         return self.q(css='section.prerequisites > .tip').visible
+
+    def click_username_dropdown(self):
+        """Click username dropdown"""
+        self.q(css='.dropdown').first.click()
+
+    @property
+    def username_dropdown_links(self):
```

Comment:
can we change the name to `username_dropdown_link_text` because it returns the text for the link.